models/Ion_auth_model.php

```diff
@@ -183,7 +183,18 @@ public function __construct()
 		$this->lang->load('ion_auth');
 
 		// initialize the database
-		$this->db = $this->load->database($this->config->item('database_group_name', 'ion_auth'), TRUE, TRUE);
+		$group_name = $this->config->item('database_group_name', 'ion_auth');
+		if (empty($group_name)) 
+		{
+			// By default, use CI's db that should be already loaded
+			$CI =& get_instance();
```

Comment:
Isn’t the CI super global already available as this here?

Reply:
I’m thinking maybe don’t override it at all with an empty group name. Or am I missing something?

Reply:
I just verified, and as I suspect, not overriding when group name is empty leads to `Call to a member function select() on null`

Reply:
This is because the `__get()` magic method from CI_Model will not be called if a `$db` member is defined in a child class.

Reply:
Ah gotcha, ok merging this then.  Thanks!

Reply:
No problem. It was good to check.